evm/src/cpu/kernel/interpreter.rs

```diff
@@ -238,9 +238,9 @@ impl<'a> Interpreter<'a> {
             0x09 => self.run_mulmod(),                                  // "MULMOD",
             0x0a => self.run_exp(),                                     // "EXP",
             0x0b => todo!(),                                            // "SIGNEXTEND",
-            0x0c => todo!(),                                            // "ADDFP254",
-            0x0d => todo!(),                                            // "MULFP254",
-            0x0e => todo!(),                                            // "SUBFP254",
+            0x0c => self.run_add_p(),                                   // "ADDFP254",
+            0x0d => self.run_mul_p(),                                   // "MULFP254",
+            0x0e => self.run_sub_p(),                                   // "SUBFP254",
```

Comment:
temporary implementations just for testing, and using a hardcoded dummy value of `101` for the prime